src/plugins/intel_gpu/src/graph/impls/ocl_v2/include/algorithm.cl

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
[random spot]
As I understand, all of these .cl includes were copied without modification. Can we reuse kernel_selector's includes to avoid potential issues caused by updating one header but missing another? This would simplify maintenance until kernel_selector is fully removed